src/example-types/filters/tagsQuery.js

```diff
@@ -80,6 +80,50 @@ let filter = ({ tags, join, field, exact }) => ({
   },
 })
 
+let result = async (node, search, schema, { options, getProvider }) => {
+  let { tags, join, field, exact, _meta, maxToCount = 20 } = node
+  if (_.size(tags) > maxToCount) {
+    return null
+  }
+
+  let provider = getProvider(node)
+
+  let results = await Promise.all(
+    _.map(
+      async tag => ({
+        tag,
+        result: await provider.runSearch(
+          options,
+          node,
+          schema,
+          provider.groupCombinator({ join: 'and' }, [
```

Comment:
We might want to port over the spark combinators utils to simplify this (https://github.com/smartprocure/spark/blob/develop/api/src/services/search/customTypes/previouslyExportedRecords.js#L52-L58) - this doesn't have to be done now, but I think we should consider promoting them to official contexture methods (maybe just in the core itself since they're not provider specific)

Reply:
no longer relevant